packages/zosfiles/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Created edit commands to edit a dataset or uss file locally [PR #1672](https://github.com/zowe/zowe-cli/pull/1672)
```

Comment:
The changelog entry can be removed from this file if nothing has changed in the zosfiles SDK in this PR 🙂